README.md

```diff
@@ -1 +1,128 @@
 # Use SIWE
+
+Use-siwe is a library that provides a react hook and a set API endpoints that
+make it dead simple to add Sign-In with Ethereum functionality to your react
+application.
+
+## Installation
+
+To install `use-siwe` and it's dependencies run the following command:
+
+```
+npm install @randombits/use-siwe iron-session
+```
+
+## Getting Started
+
+### Configure settings for `iron-session`
+
+`lib/ironOptions.ts`
+https://github.com/vvo/iron-session#ironoptions
+
+```
+import { IronSessionOptions } from 'iron-session';
+
+if (!process.env.IRON_SESSION_PASSWORD)
+  throw new Error('IRON_SESSION_PASSWORD must be set');
+
+const ironOptions: IronSessionOptions = {
+  password: process.env.IRON_SESSION_PASSWORD,
+  cookieName: 'session',
+  cookieOptions: {
+    secure: process.env.NODE_ENV === "production",
+  },
+};
+
+declare module "iron-session" {
+  interface IronSessionData {
+    address?: string | undefined;
+    nonce?: string | undefined;
+  }
+}
+
+export default ironOptions;
+```
+
+https://github.com/vvo/iron-session#typing-session-data-with-typescript
```

Comment:
Only feedback I have is this link wasn't clear to me what I should do with it. But maybe it's because I am not a developer 🤷‍♂️